README.md

```diff
@@ -53,6 +53,21 @@ remote_read:
   - url: "http://<adapter-address>:9201/read"
 ```
 
+## Environment variables
+
+All of the CLI flags are also available as environment variables, and begin with the prefix `TIMESCALE_PROMPGADAPTER`.
```

Comment:
I'd recommend something simpler(=less typing) as prefix. Eg. `TS_PROM_`?

Reply:
No problem, I'll update it.